tests/integration/004_template/main.tf

```diff
@@ -74,7 +74,7 @@ resource "time_sleep" "wait_for_all_templates" {
       # Only using depends_on doesn't work on re-apply. This makes sure the sleep happens on re-apply
       second_run = var.second_run
     }
-  create_duration = "2s"
+  create_duration = "10s"
```

Comment:
Seems like a steep uprise, maybe double it first?

Reply:
f47ac58df120f00398c91f3ecea09142c7af1f55